### PR TITLE
Cleanup license comments

### PR DESCRIPTION
**Pull request by frankduncan**
Related to #300, update license headers for most files and filetypes.  Not marking as fixed because there may still be more to do.

### CODE REVIEW

File: psm-app/cms-web/WebContent/WEB-INF/pages/provider/profile/list.jsp [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        </div>
+      </div>
+      <!-- /#mainContent -->
+
+      <h:handlebars template="includes/footer" context="${pageContext}"/>
+      <!-- #footer -->
+    </div>
+    <!-- /#wrapper -->
+  </body>
+</html>
```
Comment (PaulMorris) [inline psm-app/cms-web/WebContent/WEB-INF/pages/provider/profile/list.jsp]: Looks like this file was added here by mistake.  It was renamed from 'list.jsp' to 'profile.jsp' in a recent PR.

Comment (frankduncan) [reply psm-app/cms-web/WebContent/WEB-INF/pages/provider/profile/list.jsp]: Oh, nice catch, thanks!

**Comment by frankduncan** [comment]
Thanks @PaulMorris .  I decided the js/handlebar/css files weren't in scope (and didn't close #300 as part of this) because the reason for this change was the number of times 'enrollment' and 'ticket' were in the various comments, that I wanted to excise before attacking #485.  So I'll merge and leave open.

**Comment by PaulMorris** [comment]
Sounds good, thanks @frankduncan !

**Pull request merged**